### PR TITLE
Hw13 Spring Security ACL

### PR DESCRIPTION
**Pull request by aqus**

### CODE REVIEW

File: hw13-spring-security-acl/src/main/java/ru/otus/catalog/security/UserService.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    @Transactional
+    @Override
+    public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
+        UserInfo userInfo = userRepository.findByUsername(username)
+                .orElseThrow(() ->
+                        new UsernameNotFoundException("User with username %s is not found".formatted(username)));
+        userInfo.setLastLogin(LocalDateTime.now());
+        return new User(userInfo.getUsername(), userInfo.getPassword(), Collections.singletonList(userInfo::getRole));
```
Comment (stvort) [inline hw13-spring-security-acl/src/main/java/ru/otus/catalog/security/UserService.java:31]: Authority != Role

File: hw13-spring-security-acl/src/main/resources/data.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+insert into comments(text, book_id)
+values ('BookComment_1', 1), ('BookComment_2', 2), ('BookComment_3', 3);
+
+insert into books_genres(book_id, genre_id)
+values (1, 1),   (1, 2),
+       (2, 3),   (2, 4),
+       (3, 5),   (3, 6);
+
+insert into users(username, password, role, last_login, active)
+values ('user', '$2a$12$Q59DuP9OJUFAnYSzfsrVpusLVr/vFxQiA1JyhNfXvM3xaXUmqbT6i', 'ROLE_USER', CURRENT_TIMESTAMP, TRUE),
```
Comment (stvort) [inline hw13-spring-security-acl/src/main/resources/data.sql:20]: `ROLE_USER` как раз Authority

**Review by stvort — APPROVED** [review]

File: hw13-spring-security-acl/src/main/resources/data.sql (hunk 2 above)
Comment (aqus) [reply hw13-spring-security-acl/src/main/resources/data.sql:20]: переименовал в authority [5f3aee9](https://github.com/aqus/2023-09-otus-spring-novozhilov/pull/11/commits/5f3aee91ed3750fbd031fd9bde25c5fed28d9c05)

File: hw13-spring-security-acl/src/main/java/ru/otus/catalog/security/UserService.java (hunk 1 above)
Comment (aqus) [reply hw13-spring-security-acl/src/main/java/ru/otus/catalog/security/UserService.java]: переименовал в authority [5f3aee9](https://github.com/aqus/2023-09-otus-spring-novozhilov/pull/11/commits/5f3aee91ed3750fbd031fd9bde25c5fed28d9c05)

**Pull request merged**